src/componentsEventPage/CategoryFilter.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import styled from 'styled-components/macro'
+
+
+export default function CategoryFilter ({ onSelectFilter }) {
+  
+  return (
+     <FilterContainer>
```

Comment:
This could just be a `React.Fragment` or short `<>`

Reply:
also here, yes, I need to integrate my design :D, so it makes sense